config/job_types/ingest_journal.yml

```diff
@@ -10,6 +10,8 @@ params:
   nlp_params: dict
   do_ocr: boolean
   ojs_metadata: dict
+  image_max_width: string
```

Comment:
Wieso string und nicht int?

Reply:
Die "_init_default_params" methode in service/job/job_config.py lässt kein int zu. Weiss aber auch nicht so recht warum.
Vllt kann @scuy was dazu sagen.

Reply:
Brauchten wir bisher nicht, habe ein Ticket angelegt: http://dai-softsource.uni-koeln.de/issues/9634

Reply:
Ich würde aber dafür plädieren, die Höhe der Thumbnails nicht vom User setzen zu lassen, sondern in der Job-Config zu hinterlegen. Gleichartige Objekte sollten auch immer ähnliche Thumbnails bekommen.